ckanext/bcgov/plugin.py

```diff
@@ -50,11 +54,13 @@
                                              debug_full_info_as_list,
                                              remove_user_link,
                                              get_ofi_config,
-                                             get_ofi_resources
+                                             get_ofi_resources,
+                                             log_this
                                              )
 
 abort = base.abort
 
+log = logging.getLogger('ckan.ckanbcgov.plugin')
```

Comment:
use `'ckanext.bcgov'` instead for the logger namespace

Reply:
updated.